packages/config/src/chainConfig/networks/gnosis.ts

```diff
@@ -41,4 +40,7 @@ export const gnosisChainConfig: ChainConfig = {
   // Capella
   CAPELLA_FORK_VERSION: b("0x03000064"),
   CAPELLA_FORK_EPOCH: 648704, // 2023-08-01T11:34:20.000Z
+  // Deneb
+  DENEB_FORK_VERSION: b("0x04000064"),
+  DENEB_FORK_EPOCH: Infinity,
```

Comment:
This change is to safe guard against previous issue
- https://github.com/ChainSafe/lodestar/issues/5322

See fix for the issue
- https://github.com/ChainSafe/lodestar/pull/5324